_includes/meetups.html

```diff
@@ -4,19 +4,29 @@ <h1>EDICIONES PASADAS</h1>
     <p>Conocé el contenido de las ediciones pasadas.</p>
   </header>
 
-  <ul>
-    {% for meetup in site.meetups reversed %}
-      <li class="meetup-event">
-        <a href="{{ meetup.url }}" class="meetup-event__link">
-          <span class="meetup-event__date">
-            {{ meetup.date | date_to_string }}
-          </span>
-          <span class="meetup-event__company">
-            {{ meetup.title }}
-            <img src="/assets/images/arrow.svg" aria-hidden="true">
-          </span>
-        </a>
-      </li>
-    {% endfor %}
-  </ul>
+  {%- assign meetups_by_year = site.meetups
+        | group_by_exp: "m", "m.date | date: '%Y'"
+        | sort: "name"
+        | reverse
+  -%}
+
+  {%- for year in meetups_by_year -%}
+    <h2 class="meetup-event__date_year">{{ year.name }}</h2>
+
+    <ul>
+      {%- for meetup in year.items -%}
```

Comment:
can we reverse this list so that the older dates go below the newer?

![image](https://github.com/user-attachments/assets/16be201b-ffe0-4391-9c82-9b3d99b4ddc1)

Reply:
![a man in a purple shirt is making a face](https://media.tenor.com/jldNUqYT_K4AAAAC/brian-regan-derp.gif)
Thank you <3